docs/updating-spot-retries.md

```diff
@@ -0,0 +1,93 @@
+(spot-retry-page)=
+
+# Spot instance failures and retries
+
+This page describes changes in how Nextflow handles spot instance failures on AWS and Google Cloud, the impact of those changes, and how to configure spot retry behavior for your pipelines. It applies to Nextflow 24.10 and later.
+
+## Retry behavior
+
+Up to version 24.10, Nextflow silently retries spot instance failures up to `5` times when using AWS Batch or Google Batch. These retries were controlled by cloud-specific configuration parameters (e.g., `aws.batch.maxSpotAttempts`) and happened in cloud infrastructure without explicit visibility to Nextflow.
```

Comment:
Since the rest of this paragraph is in past tense, it makes sense to apply the same to the first sentence. 

```suggestion
Up to version 24.10, Nextflow would silently retry spot instance failures up to `5` times when using AWS Batch or Google Batch. These retries were controlled by cloud-specific configuration parameters (e.g., `aws.batch.maxSpotAttempts`) and happened in cloud infrastructure without explicit visibility to Nextflow.
```